### ISSUE TITLE
[BUG] Can't create PrometheusRule in CRC 1.18.0 

### ISSUE THREAD

**Issue by alanconway** [code-ready/crc#1648]
### General information

  * OS: Linux Fedora 32
  * Hypervisor: KVM
  * Did you run `crc setup` before starting it? Yes
  * Running CRC on: Laptop

## CRC version
```bash
CodeReady Containers version: 1.18.0+bb304aa
OpenShift version: 4.6.1 (embedded in binary)
```
NOTE: This is a regression from earlier CRC versions, verified OK on 1.17.0 and 1.16.0.
  
## CRC status
```bash
DEBU CodeReady Containers version: 1.18.0+bb304aa 
DEBU OpenShift version: 4.6.1 (embedded in binary) 
Checking file: /home/aconway/.crc/machines/crc/.crc-exist
DEBU About to run SSH command with hidden output  
DEBU SSH command succeeded                        
DEBU About to run SSH command:
df -B1 --output=size,used,target /sysroot | tail -1 
DEBU SSH command results: err: <nil>, output: 32720793600 13695254528 /sysroot 
CRC VM:          Running
OpenShift:       Running (v4.6.1)
Disk Usage:      13.7GB of 32.72GB (Inside the CRC VM)
Cache Usage:     41.01GB
Cache Directory: /home/aconway/.crc/cache
```

## CRC config
```bash
- cpus                                  : 8
- memory                                : 16384
- pull-secret-file                      : /home/aconway/cluster/etc/pull-secret
- skip-check-libvirt-running            : true
```

## Host Operating System
```bash
NAME=Fedora
VERSION="32 (Workstation Edition)"
ID=fedora
VERSION_ID=32
VERSION_CODENAME=""
PLATFORM_ID="platform:f32"
PRETTY_NAME="Fedora 32 (Workstation Edition)"
ANSI_COLOR="0;34"
LOGO=fedora-logo-icon
CPE_NAME="cpe:/o:fedoraproject:fedora:32"
HOME_URL="https://fedoraproject.org/"
DOCUMENTATION_URL="https://docs.fedoraproject.org/en-US/fedora/f32/system-administrators-guide/"
SUPPORT_URL="https://fedoraproject.org/wiki/Communicating_and_getting_help"
BUG_REPORT_URL="https://bugzilla.redhat.com/"
REDHAT_BUGZILLA_PRODUCT="Fedora"
REDHAT_BUGZILLA_PRODUCT_VERSION=32
REDHAT_SUPPORT_PRODUCT="Fedora"
REDHAT_SUPPORT_PRODUCT_VERSION=32
PRIVACY_POLICY_URL="https://fedoraproject.org/wiki/Legal:PrivacyPolicy"
VARIANT="Workstation Edition"
VARIANT_ID=workstation
```

### Steps to reproduce

  1. crc setup; crc start; crc console # log in as kubeadmin
  2. In console OperatorHub: install Red Hat Cluster Loggging Operator
  3. Start logging and log forwarder, don't worry about the bogus IP addres.
```bash
oc create -f - <<EOF
---
apiVersion: "logging.openshift.io/v1"
kind: "ClusterLogForwarder"
metadata:
  name: "instance"
  namespace: "openshift-logging"
spec:
  pipelines:
    - inputRefs: [ application, infrastructure, audit ]
      outputRefs: [ ff ]
  outputs:
    - name: ff
      type: fluentdForward
      url: tcp://bogus.example.com
---
apiVersion: "logging.openshift.io/v1"
kind: "ClusterLogging"
metadata:
  name: "instance"
  namespace: "openshift-logging"
spec:
  managementState: "Managed"
  collection:
    logs:
      type: "fluentd"
      fluentd:
        resources: {}
EOF
```
  4. oc logs -n openshift-logging -l name=cluster-logging-operator

### Expected

Operator should start sucessfully.

### Actual

Operator looping on this error message, failing to create a prometheus rule:

```
{"level":"error","ts":1604687905.1160727,"logger":"controller-runtime.controller","msg":"Reconciler error","controller":"clusterlogforwarder-controller","request":"openshift-logging/instance","error":"Unable to reconcile collection for \"instance\": failure creating the fluentd PrometheusRule: Internal error occurred: failed calling webhook \"prometheusrules.openshift.io\": Post \"https://prometheus-operator.openshift-monitoring.svc:8080/admission-prometheusrules/validate?timeout=5s\": no endpoints available for service \"prometheus-operator\"","stacktrace":"github.com/go-logr/zapr.(*zapLogger).Error\n\t/go/src/github.com/openshift/cluster-logging-operator/vendor/github.com/go-logr/zapr/zapr.go:128\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler\n\t/go/src/github.com/openshift/cluster-logging-operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:258\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem\n\t/go/src/github.com/openshift/cluster-logging-operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:232\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).worker\n\t/go/src/github.com/openshift/cluster-logging-operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:211\nk8s.io/apimachinery/pkg/util/wait.BackoffUntil.func1\n\t/go/src/github.com/openshift/cluster-logging-operator/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:155\nk8s.io/apimachinery/pkg/util/wait.BackoffUntil\n\t/go/src/github.com/openshift/cluster-logging-operator/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:156\nk8s.io/apimachinery/pkg/util/wait.JitterUntil\n\t/go/src/github.com/openshift/cluster-logging-operator/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:133\nk8s.io/apimachinery/pkg/util/wait.Until\n\t/go/src/github.com/openshift/cluster-logging-operator/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:90"}
```

### Logs

Before gather the logs try following if that fix your issue
```bash
$ crc delete -f
$ crc cleanup
$ crc setup
$ crc start --log-level debug
```

Please consider posting the output of `crc start --log-level debug`  on http://gist.github.com/ and post the link in the issue.

**Comment by alanconway** [comment]
> Please consider posting the output of crc start --log-level debug on http://gist.github.com and post the link in the issue.

https://gist.github.com/alanconway/d3acf957e91ef5cca34a07bdd0f9643b

**Comment by cfergeau** [comment]
Have you enabled the monitoring stack in crc before installing/starting your operator? https://code-ready.github.io/crc/#starting-monitoring-alerting-telemetry_gsg

**Comment by noelo** [comment]
I'm seeing the same issue on 1.19 --> https://github.com/code-ready/crc/issues/1712

**Comment by praveenkumar** [comment]
https://github.com/code-ready/crc/issues/1704 is tracking for this one and have a manual instructions to enable it, should be fixed in next release (1.20.0).
